wandb/sdk/launch/runner/kubernetes.py

```diff
@@ -56,6 +61,19 @@ def get_status(self) -> Status:
             name=self.name, namespace=self.namespace
         )
         status = job_response.status
+        try:
+            pod_status = self.core_api.read_namespaced_pod_log(name=self.pod_names[0], namespace=self.namespace)
```

Comment:
why `read_namespaced_pod_log` and not `read_namespaced_pod_status`?

Reply:
The log issues an error until the container is actually running. While the status would need to be parsed to determine the nature of the failure mode. This is a way to handle any sort of errors (for example image pulling)